provisioning/requirements.yml

```diff
@@ -1,6 +1,7 @@
 - src: https://github.com/idi-ops/ansible-facts
   name: facts
 
-- src: https://github.com/gtirloni/ansible-role-nodejs
+- src: git+https://github.com/gtirloni/ansible-role-nodejs.git
```

Comment:
With @gtirloni's fix for [GPII-2060](https://issues.gpii.net/browse/GPII-2060), a comment is added to explain the work around.